docs/build_parameters.rst

```diff
@@ -244,7 +244,7 @@ example, if **reactor_config** defines::
     - kojisecret
     - pulpsecret
 
-Secrets named **kojisecret** and **pulpsecret** must be available in orchestrator and
+Secrets named **kojisecret** must be available in orchestrator and
```

Comment:
Now there's just one of them there, so this should probably read `A secret named ...`, and further down, `It doesn't need ...`.

Reply:
Fixed.